src/containers/community/CommunityEditor.js

```diff
@@ -36,6 +37,7 @@ const categories = {
   let validating = some(communityValidation.pending)
   let { community, errors } = communityEditor
   let saving = pending[CREATE_COMMUNITY]
+  let uploadingImage = !!pending[UPLOAD_IMAGE]
```

Comment:
oh actually that's fine too. we could just call it 'pending' to indicate that it's not just because saving is taking place